.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "eslint.workingDirectories": ["tests"]
```

Comment:
shouldnt be here no?

Reply:
what do you mean? like gitignored ? this is useful for all vscode users I guess